presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -146,6 +146,10 @@ Property Name                                      Description
                                                    ignored. This is equivalent to the
                                                    ``hive.mapred.supports.subdirectories`` property in Hive.
 
+``hive.ignore-absent-partitions``                  Ignore partitions when the file system location does not     ``false``
+                                                   exist rather than failing the query. This skips data that
```

Comment:
nit, please feel free to ignore: 
* add a "," after exist. 
* line "This skips data that may be expected to be part of the table" does not seem to be adding more context.

Reply:
@ankitdixit it's exactly as @electrum requested :)